### PR TITLE
@types/dwt: Add Load and AutoLoad to WebTwainEnv to allow for manual loading

### PR DESCRIPTION
**Pull request by jbh**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: http://developer.dynamsoft.com/dwt/autoload
- [x] Increase the version number in the header if appropriate.

This allows the user to set AutoLoad to false and then Load() Dynamsoft when needed.

### CODE REVIEW

**Review by yushulx — APPROVED** [review]
Thanks for your changes

**Review by yushulx — APPROVED** [review]

**Review by yushulx — APPROVED** [review]

**Review by yushulx — APPROVED** [review]

File: types/dwt/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,7 @@
-// Type definitions for dwt 12.3
+// Type definitions for dwt 13.1
```
Comment (vvakame) [inline types/dwt/index.d.ts]: Could you make v12 folder and copy old files to that dir?
please see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

Comment (vvakame) [inline types/dwt/index.d.ts]: Could you make v12 folder and copy old files to that dir?
please see https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version

**Review by vvakame — CHANGES_REQUESTED** [review]

File: types/dwt/index.d.ts (hunk 1 above)
Comment (jbh) [reply types/dwt/index.d.ts]: Sure will. Figured I'd miss one part! :)

**Pull request merged**